### ISSUE TITLE
Fix non-standard of 'struct' in VCD traces

### ISSUE THREAD

**Issue by AnikBalo** [verilator/verilator#4227]
Verilator version: 4.110
OS: Ubuntu-20.04
I performed RTL simulation using verilator and generated a VCD of that RTL [ https://github.com/openhwgroup/cva6 ]  simulations without any error. While using that VCD in cadence tools, the tool saying there is syntax error at line 114 in VCD file.

The VCD file snapshot: 
![vcd_snapshot](https://github.com/verilator/verilator/assets/96606368/ad7cb2cf-ee63-4ed2-8aed-cc26c3882bd7)

Can anyone help what's wrong here? 
Thanks!

**Comment by wsnyder** [comment]
"struct" isn't legal in VCD's it needs to be "module".  Perhaps you could create a pull request to add this to e.g. t_trace_complex.v and fix verilated_vcd.cpp to only print "module"?

**Comment by atorregrosa-smd** [comment]
Actually, this was introduced in #2776 & #2805, as [GTKWave parses it that way](https://github.com/gtkwave/gtkwave/blob/master/gtkwave3-gtk3/src/helpers/vcd2fst.c#L887) to show a struct icon in the hierarchy tree. 

The LRM does however say that the only supported scope types are `module`, `task`, `function`, `begin` and `fork`, so this relies on non-standard behavior. It's probably better then to leave the gtkwave-specific types just in `.fst` waves.

**Comment by wsnyder** [comment]
@atorregrosa-smd, agreed, would you be willing to make a pull request to fix this please?

**Comment by atorregrosa-smd** [comment]
Yes, no problem.
